impedance/tests/test_circuit_elements.py

```diff
@@ -36,8 +36,8 @@ def test_all():
     inputs = {'R': 1,
               'C': 1,
               'L': 1,
-              'W': 2,
-              'A': 1,
+              'Wo': 2,
+              'W': 1,
               'E': 2,
               'G': 2,
               'T': 4,
```

Comment:
should probably delete the test for RR too

Reply:
Do you mean remove RR tests? I think Matt took out RR since it's a duplicate element.

Reply:
@nealde I took the `RR` element out because I thought it was just a placeholder for implementing the multiple character elements! Should we keep it? If you think it makes sense I can add it to the tests, but I think it's potentially confusing to people to leave it in `circuits.elements`.

Reply:
ah - yeah, I meant don't forget to remove the RR tests, it looked like they were still in there. It was totally just a placeholder

Reply:
ah ok, it looks like they have been removed in the current state of the PR.  When I saw this file before, they hadn't been marked as deleted!